README.md

```diff
@@ -123,6 +123,7 @@ The full API Reference is available here.
 * [`removeBucket`](https://docs.min.io/docs/javascript-client-api-reference#removeBucket)
 * [`listObjects`](https://docs.min.io/docs/javascript-client-api-reference#listObjects)
 * [`listObjectsV2`](https://docs.min.io/docs/javascript-client-api-reference#listObjectsV2)
+* `listObjectsV2WithMetadata` (Extension)
```

Comment:
The doc reference should be similar to `listObjectsV2` with a hyperlink, relevant docs should be added in docs/API.md

Reply:
This should do it